.github/workflows/docker.yaml

```diff
@@ -0,0 +1,62 @@
+name: Publish Docker
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag for the Docker image (optional)'
+        required: false
+  create:
```

Comment:
I think it's `push`? 
```suggestion
  push:
```

Reply:
`create` worked fine in the original project.